src/api/clinical/complaint/complaint.controller.ts

```diff
@@ -81,9 +80,9 @@ export class ComplaintController extends BaseController{
     search = async (request: express.Request, response: express.Response): Promise<void> => {
         try {
 
-            const filters = await this._validator.search(request);
-            await this.authorizeUser(request, filters);
-            const searchResults = await this._service.search(filters);
```

Comment:
Why the pattern for the search broken here?

Reply:
This endpoint is not like generic search, It's like search by patient user id.